### DISCUSSION TITLE
rAudio • 20220814: Tag Editor does not work

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#990] [Fixed Issues]
Selecting the "Tag Editor" is a no-op.

Version: rAudio • 20220814
Kernel: 5.15.56-3 • aarch64
Hardware: Raspberry Pi 3B Rev 1.2

**Comment by klassikmann** [comment]
Can anyone second or debunk my observation?

**Reply by R101G** [reply]
Yes, I tried using the editor for the first time a couple of weeks ago and found that I could display the data but not alter it. As I was not sure how it was supposed to work, I did not report it.

**Reply by rern** [reply]
Please try the UPDATE branch.

**Reply by klassikmann** [reply]
The UPDATE branch solved the problem.
